src/test/regress/checkinc.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python
 
 import sys, os, re, subprocess
+try:
+    import sysconfig
+    multiarch_triplet = sysconfig.get_config_var('MULTIARCH')
+except ImportError:
+    multiarch_triplet = sysconfig.get_config_var('MULTIARCH')
```

Comment:
the code here is wrong... You can define `multiarch_triplet=''` here or try import `distutils.sysconfig` inside and use `distutils.sysconfig.get_config_var('MULTIARCH')` instead. Since HAWQ is near new release, we will fix it for you and remain your name. @cbaenziger